packages/dds/matrix/src/test/memory/sharedMatrix.spec.ts

```diff
@@ -9,20 +9,20 @@ import { SharedMatrix } from "../../index.js";
 import { createLocalMatrix } from "../utils.js";
 
 describe("SharedMatrix memory usage", () => {
+	// The value to be set in the cells of the matrix.
 	const matrixValue = "cellValue";
 	// The test matrix's size will be 10*10, 100*100, 1000*1000.
 	const matrixSizes = [10, 100, 1000];
 
 	// The number of operations to perform on the matrix.
 	const operationCounts = [10, 100, 1000];
 
-	beforeEach(async () => {});
-
-	afterEach(() => {});
-
 	for (const matrixSize of matrixSizes) {
 		describe(`Size of ${matrixSize}*${matrixSize} SharedMatrix`, () => {
-			for (const count of operationCounts) {
+			// Filter counts to ensure they do not exceed matrixSize
+			const validCounts = operationCounts.filter((count) => count <= matrixSize);
```

Comment:
This should only apply to removal tests. `count <= matrixSize` is valid and useful for insertion test cases, etc.

Reply:
Agreed. Isolate the remove test cases to sole on the filtered-count for-loop.